### ISSUE TITLE
canary version

### ISSUE THREAD

**Issue by kingdonb** [teamhephy/workflow#85]
A canary is an image that is automatically built from the latest HEAD in the Master branch of every [teamhephy GitHub component's](//github.com/teamhephy) git project repo.  Without incrementing any version numbers, it should be possible to start a cluster that uses canary images, which has all of the patches that would go into the next `deisrel` release, even if it breaks something.

This will make it easier to design our CI and rebuild our e2e conformance suite.

~~https://docs.teamhephy.info/managing-workflow/tuning-component-settings/#customizing-the-controller~~

```
the image used to run buildpack application slugs (default: "quay.io/hephyci/slugrunner:canary")
```

~~This isn't true, is it?~~

~~I have:~~

```
  image: hephy/slugrunner:v2.6.1
```

~~from `kd get cm -o yaml slugrunner-config` (`alias kd='kubectl --namespace=deis'`)~~
The #88 PR addressed the docs issue.

This ticket will stand open until we've got Canary builds in place.

**Comment by Cryptophobia** [comment]
Yes, we keep all of our images in docker hub hephy org.

**Comment by kingdonb** [comment]
After this, and #84, we will need to roll the deployments for docs sites.

I'll address them both in one PR.

**Comment by Cryptophobia** [comment]
Thanks a lot for addressing this @kingdonb !

**Comment by kingdonb** [comment]
Actually, what I was thinking when I opened this was we don't have a proper canary version

(Building up the CI a little bit might be necessary first, to close this one.)

I included a change in patch-2 branch PR, #88, to address the docs issue that started this ticket.

**Comment by Cryptophobia** [comment]
Closed by #88
